server/server.go

```diff
@@ -877,6 +977,96 @@ func (s *Server) forwardPollRequest(v *visitor, m *message) {
 	}
 }
 
+func (s *Server) publishToWebPushEndpoints(v *visitor, m *message) {
+	subscriptions, err := s.webPushSubscriptionStore.GetSubscriptionsForTopic(m.Topic)
+
```

Comment:
1. remove new line
2. Idiomatic Go doesn't have "Get" Prefix, so it'd be `webPush.SubscriptionsForTopic(m.Topic)` here